include/net/net_stats.h

```diff
@@ -228,6 +228,27 @@ struct net_stats_ipv6_mld {
 	net_stats_t drop;
 };
 
+#if !defined(NET_TC_COUNT)
```

Comment:
Shouldn't this be always set to, at least, 1?

Reply:
It is set to 1 in net_core.h, but I had some issues with this earlier. I will check if this check can be removed from net_stats.h